searx/settings.yml

```diff
@@ -1347,6 +1347,27 @@ engines:
       require_api_key: false
       results: html
 
+  - name : whaleslide
+    shortcut : whale
+    engine : json_engine
+    paging : True
+    # This API Token is needed to work
+    search_url : https://search.whaleslide.com/api/v1/search_v2/web/{pageno}?q={query}&api_token=f8OhUDEYKUICHDnIxEgI7Cb4uYyTBqT4nO8iueNbfTO3devS24yElGqM7nCm
```

Comment:
Is this your development token? How can others acquire their own tokens? In general I am not a huge fan of hardwiring tokens into searx codebase. I would rather let instance admins get their own tokens and set it for themselves.

Reply:
**How can others acquire their own tokens?**

> The API used here is required and can be found with Developer Optiones > Network Tab

No not my Token, they seem to use it everytime.

Maybe different users will get a different one but this has worked for me.
Its the same today as the one from weeks ago.